includes/dashboard/showSelectedGroup.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/* 
+ * this is to show the selected group of the users, selected by the admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+
+if ( ! isset($_POST['groupId']) )
+{
+	echo "Sorry, there is some problem";
+	die ();
+}
+
+$groupID = rtrim($_POST['groupId'], ';');
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
```

Comment:
__[code_sniffer]__
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Space found before comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->